Content.Server/Body/Systems/BodySystem.cs

```diff
@@ -65,6 +65,58 @@ private void OnApplyMetabolicMultiplier(
         }
     }
 
+    // Begin DeltaV - Have body appearances be based on what parts they actually have on spawn
+    protected override void MapInitAppearance(Entity<BodyComponent> ent)
```

Comment:
no reason this cant be in shared + put it in a _DV partial system instead of directly in SharedBodySystem

Reply:
subscribed to what event exactly? (BodyComponent, MapInitEvent) are already being used by this server system

Reply:
not subscribed just have this method in the partial system and call it